File: src/CriminalContact.Entities/Player.cs
namespace CriminalContact.Entities
{
    public class Player
    {
        public Player(string name)
        {
            Name = name;
            IsDead = false;
        }

        public Account BankAccount { get; set; }

        public bool IsDead { get; }

        public string Name { get; }

        public Player Parent { get; set; }

        public PlayerRole Role { get; set; }
    }
}

File: src/app/models/player.ts
import { Account } from "./account";

export class Player {
    //private readonly _account: Account;

    constructor(
        private _name: string,
        private _account: Account
    ) {        
    }

    get name(): string {
        return this._name;
    }

    get account(): Account {
        return this._account;
    }
}